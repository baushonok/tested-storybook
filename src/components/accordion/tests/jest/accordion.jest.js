import React from "react";
import renderer from "react-test-renderer";

import Accordion from "../..";

test("Accordion should exist", () => {
  const component = renderer.create(<Accordion />);

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
