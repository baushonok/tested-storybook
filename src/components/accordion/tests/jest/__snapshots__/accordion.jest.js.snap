// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Accordion should exist 1`] = `
<button
  onClick={[Function]}
  type="button"
>
  Click me
</button>
`;
